src/core/plugins/plugins-reducer.js

```diff
@@ -44,6 +44,14 @@ export const plugins = (
       }
       for (const pluginId of Object.keys(action.payload)) {
         const plugin = action.payload[pluginId]
+
+        // Don't stop loading the bundle if plugin(s) fail to load. Some plugins may rely on advanced features
+        // that aren't locally available so we can skip loading those but should continue and load what we can.
+        if (plugin == null) {
+          delete out.init[pluginId]
```

Comment:
This is Redux, so we can't just delete things. I think we can do `out.init = { ...out.init, [pluginId]: false }`, though.